homeassistant/components/tedee/coordinator.py

```diff
@@ -82,6 +84,15 @@ async def _async_get_bridge() -> None:
             ", ".join(map(str, self.tedee_client.locks_dict.keys())),
         )
 
+        if not self._current_locks:
+            self._current_locks = set(self.tedee_client.locks_dict.keys())
```

Comment:
We don't need `.keys()` to copy the keys of a dict to a set. The keys are copied by default.

Reply:
good catch! Added to https://github.com/home-assistant/core/pull/106994 which extends this functionality anyways.